src/dkg_hybriddkg.erl

```diff
@@ -112,13 +115,42 @@ init(Id, N, F, T, G1, G2, Round, Options) ->
          leader_cap = leader_cap(1, N),
          shares_map = Shares}.
 
+%% the dgk starts to go on its own, this is here for the sake of
+%% fakecast for now, since all the other protocols take input.
+input(State, _) ->
+    start(State).
+
 start(DKG = #dkg{id=Id, u=G1}) ->
     #{Id := MyShares} = SharesMap = DKG#dkg.shares_map,
     Secret = erlang_pbc:element_random(erlang_pbc:element_new('Zr', G1)),
     {NewShares, {send, ToSend}} = dkg_hybridvss:input(MyShares, Secret),
     {DKG#dkg{shares_map = SharesMap#{Id => NewShares}},
      {send, dkg_util:wrap({vss, Id}, ToSend)}}.
 
+handle_msg(DKG=#dkg{await_vss = true}, Sender, {{vss, SharesId}, SharesMsg}) ->
```

Comment:
I noticed while running the tests locally that this clause is pretty much uncovered?

Reply:
I'll do my best to extend coverage here later, but at the moment I'm not sure it's worth the effort for such a rare case.

Reply:
Fair enough, I know triggering this particular branch is probably a lot trickier as we discussed offline